packages/plugins/plugin-deck/src/layout.ts

```diff
@@ -15,9 +15,18 @@ export const openEntry = (deck: string[], entryId: string, options?: OpenLayoutE
       return;
     }
 
+    const key = options?.key;
     const plankPositioning = options?.positioning ?? 'start';
     const pivotId = options?.pivotId;
 
+    if (key) {
+      const index = draft.findIndex((id) => id.startsWith(key));
+      if (index !== -1) {
+        draft.splice(index, 1, entryId);
```

Comment:
what does draft mean?

Reply:
@ZaymonFC wrote this util is using `immer`, my understanding is that `draft` is a clone of the incoming value that can be mutated and then at the end it will be returned as the new value